src/metabase/models/card.clj

```diff
@@ -45,6 +46,63 @@
   (db/cascade-delete! 'CardLabel :card_id id))
 
 
+;;; ## ---------------------------------------- PERSISTENCE FUNCTIONS ----------------------------------------
+
+
+(defn create-card
```

Comment:
Hate to sound like a broken record but these should really end in a bang (`!`). See the rational in PR #2759